src/billing/api/index.ts

```diff
@@ -120,5 +150,43 @@ export const getInvoices = (): ReturnType<typeof getBillingInvoices> => {
     },
   ]
 
-  return makeResponse(200, invoices)
+  return makeResponse(200, invoices, 'getInvoices')
+}
+
+export const makeCheckoutPayload = (data: any): CheckoutRequest => {
+  const {
+    shouldNotify,
+    notifyEmail,
+    balanceThreshold,
+    paymentMethodId,
+    country,
+    intlSubdivision,
+    usSubdivision,
+    street1,
+    street2,
+    city,
+    postalCode,
+  } = data
+
+  return {
+    notify: shouldNotify,
+    notifyEmail,
+    balanceThreshold,
+    paymentMethodId,
+    country,
+    subdivision: usSubdivision ?? intlSubdivision,
+    street1,
+    street2,
+    city,
+    postalCode,
+    status: RemoteDataState.Done,
+  }
+}
+
+export const postCheckoutInformation = async (
+  data: any
+): Promise<PostAccountUpgradeResult> => {
```

Comment:
We can try and get the return type of the function like the other functions here to help maintain a strong relationship with the API contract